### ISSUE TITLE
Pose positions wont save & return to default after DUMP & configuring notecard

### ISSUE THREAD

**Issue by djxpurity** [AVsitter/AVsitter#143]
I am using Firestorm 64 bit viewer (up to date) using Windows 11
After entering in my poses and using the helper to adjust 3 different sitters, I click save on all of the new adjustments. When I click on DUMP, I enter in the correct values into the notecard and save it. Bit when I sit on the object again, all of the sitters return to the default positions and poses are default as well. 

Proof I have the correct contents
https://prnt.sc/Rg57bmdIKuh3

Notecard configuration
https://prnt.sc/tLq0M82v5j2I

Screenshot of what happens when I sit after applying positions
https://prnt.sc/_gl-kI5yNzv8

No clue why this is happening. I got scripts from the official github

**Comment by selticq** [comment]
It is a bug in the dump process. As you can see, there is a | between POSE and the pose name. It should be a space. Replace | with spaces and it will work fine.

**Comment by nothingisavailableatall** [comment]
I am having exactly the same problem.

**Comment by Sei-Lisa** [comment]
This looks like a duplicate of #147, i.e. a problem importing the scripts. Please look there for a solution.

**Comment by nothingisavailableatall** [comment]
I got the scripts from here, the latest hotfix 2.2-03.01
I copied and pasted the names to create the scripts then copied and pasted the contents into the scripts.  
What could go wrong?  I've followed the instructions to the letter, watched the video, started over 3 times.  
  I don't know where else to look.

Everything seems to work while I'm setting it up.  But after I copy the dump (from the website) and it does it's finalizataion, when I sit on the chair, it's the same as the first time I sat on it before setting up the poses.

**Comment by nothingisavailableatall** [comment]
Ok, I figured out the problem.  This is what the DUMP put on the website:

~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
--✄--COPY BELOW INTO "AVpos" NOTECARD--✄--

"CUSHIONMESH" AVsitter 2.2

POSE|Female1|HDstoolF04
POSE|Female2|HDstoolF05
POSE|Female3|HDstoolF06
{Female1}<0,0.499,-0.069><-90,0,-90>
{Female2}<0,0.499,-0.069><-90,0,-90>
{Female3}<0,0.499,-0.199><-90,0,-90>

--✄--COPY ABOVE INTO "AVpos" NOTECARD--✄--



end
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

On these 3 lines ...
POSE|Female1|HDstoolF04
POSE|Female2|HDstoolF05
POSE|Female3|HDstoolF06

Note the pipe between POSE and the name of the pose.  I removed that pipe and replaced it with a space and everything works.  Hopefully, this will help the next person who follows the instructions to the letter and it just doesn't work.

**Comment by Sei-Lisa** [comment]
Maybe it's a problem in more recent versions of Windows, I don't know because I don't use Windows. The fix in the referred post was to use Firestorm's Import feature to import the script, rather than copy and paste.

If this becomes more of a problem, the constant could be assigned at runtime using llUnescapeURL or similar.

**Comment by nothingisavailableatall** [comment]
I use Firestorm.  The extra pipe was there in the chat dump as well.  The scripts are fine, it's the DUMP text that is causing the problem.  Here is the dump text from chat:

[09:47] ◆--✄--COPY BELOW INTO "AVpos" NOTECARD--✄--
[09:47] ◆
[09:47] ◆"CUSHIONMESH" AVsitter 2.2
[09:47] ◆
[09:47] ◆POSE|Female1|HDstoolF04
[09:47] ◆POSE|Female2|HDstoolF05
[09:47] ◆POSE|Female3|HDstoolF06
[09:47] ◆POSE|Male1|HDstoolM01
[09:47] ◆POSE|Male2|HDstoolM02
[09:47] ◆POSE|Male3|HDstoolM04
[09:47] ◆{Female1}<0,0.499,-0.069><-90,0,-90>
[09:47] ◆{Female2}<0,0.499,-0.069><-90,0,-90>
[09:47] ◆{Female3}<0,0.499,-0.199><-90,0,-90>
[09:47] ◆{Male1}<0.037,0.499,-0.093><-95.4,0,-90>
[09:47] ◆{Male2}<0.037,0.499,-0.093><-95.4,0,-90>
[09:47] ◆{Male3}<0.047,0.519,-0.172><-97.7,-0,-90>
[09:47] ◆
[09:47] ◆--✄--COPY ABOVE INTO "AVpos" NOTECARD--✄--
[09:47] ◆

**Comment by Sei-Lisa** [comment]
No, the scripts are not fine; if they were fine the dump would work correctly. Open [AV]adjuster and look at line 50, where it says:

    string SEP = " "; // OSS::string SEP = "\u007F";

That's a space, and it shouldn't be a space. Try to use the Firestorm Import feature to re-import [AV]adjuster and the dump will work fine.
